### PR TITLE
Use consistent phrasing for all macro summaries

### PR DESCRIPTION
**Pull request by benesch**

### CODE REVIEW

File: src/libstd/macros.rs [hunk 1]
```diff
@@ -210,8 +210,9 @@ macro_rules! eprintln {
     })
 }
 
-/// A macro for quick and dirty debugging with which you can inspect
-/// the value of a given expression. An example:
+/// Prints the value of a given expression for quick and dirty debugging.
```
Comment (Centril) [inline src/libstd/macros.rs]:
```suggestion
/// Prints and returns back the value of a given expression for quick and dirty debugging.
```

Your change gives the impression that the macro just prints and is typed at `()` but the macro returns back the value passed. I think an essential point has been lost here.

Comment (benesch) [reply src/libstd/macros.rs]: Good point, done.

**Review by Centril — APPROVED** [review]
Looks great; r=me with green travis.

**Pull request merged**